specs/git/rebase_master_into_current_branch.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: Set master as upstream and rebase master into feature branch
+command: |-
+    git branch -u master
+    git checkout master
+    git pull
+    git checkout {{featurebranch}}
+    git pull --rebase
+tags:
```

Comment:
Thoughts on trying to set the upstream back to origin/$FEATURE_BRANCH ?

Reply:
I'm thinking that we don't need to necessarily change the upstream at all. We could just write `git pull origin master --rebase` instead.